ansible/roles/fews/files/glossis_workflow.yml

```diff
@@ -119,3 +131,27 @@ spec:
       env:
       - name: GOOGLE_APPLICATION_CREDENTIALS
         value: "/root/creds/google_credentials.json"
+
+  - name: glossis-vtk
+    inputs:
+      parameters:
+      - name: argument
+      - name: volume
+    container:
+      imagePullPolicy: Always
+      image: deltares/dgds-glossis:vtk
+      command: ["python"]
+      args: ["/root/glossis.py", "dgds-data", "fews_glossis/", "projects/dgds-gee/glossis-test", "gee-test", "--{{inputs.parameters.argument}}"]
```

Comment:
Wouldn't it be an improvement if all these things have commandline flags instead of bare arguments? Like `--project dgds-data --input-folder fews_glossis`

Reply:
Yes good idea